README.md

```diff
@@ -131,37 +131,6 @@ struct Foo {
 }
```

Comment:
Lines 118 and 129 need to be fixed to remove `$value` and probably document the new way instead.

Line 120 needs to be removed as no longer relevant.

Reply:
Fixed